sox.features.info.json

```diff
@@ -108,43 +109,43 @@
         }, {
             "name": "spoilerTip",
             "desc": "Differentiate spoilers from empty blockquotes",
-            "meta": "http://meta.stackexchange.com/questions/104085/differentiate-spoilers-from-empty-block-quotes",
-            "match": "*://*/questions*,*://*/review*",
-            "exclude": "SE1.0"
+            "meta": "https://meta.stackexchange.com/q/104085",
+            "match": "*://*/questions/*,*://*/review*",
+            "exclude": "SE1.0,*://*/questions*tab=*,*://chat.*.com/*"
         }, {
             "name": "standOutDupeCloseMigrated",
             "desc": "Add highlighted tags to closed/on hold/duplicate/migrated questions on question lists",
             "extended_description": "Adds a coloured box at the end of a title (that replaces the standard [duplicate], etc.) in question lists to more easily tell what the state of a question is",
-            "meta": "http://meta.stackexchange.com/questions/257021/proposal-to-make-duplicate-closed-and-migrated-in-the-title-more-obvious",
-            "match": "*://*.com/,*://*.com/questions,*://*.com/questions/tagged/*",
+            "meta": "https://meta.stackexchange.com/q/257021",
+            "match": "*://*.com/,*://*.*/questions*",
             "exclude": "*://chat.*.com/*,SE1.0,*://*/review*",
             "feature_packs": ["major_ui, key_feature", "power_user"],
             "usesApi": true
         }, {
             "name": "tabularReviewerStats",
             "desc": "Display reviewer stats on /review/suggested-edits in table form",
-            "meta": "http://meta.stackexchange.com/q/276946/260841",
+            "meta": "https://meta.stackexchange.com/q/276946",
             "match": "*://*.com/review/suggested-edits/*",
             "exclude": "",
             "feature_packs": ["power_user"]
         }, {
             "name": "topAnswers",
             "desc": "Improve answer visibility by listing top answers",
             "meta": "",
-            "match": "*://*/questions*",
-            "exclude": "SE1.0,*://*/questions/ask"
+            "match": "*://*/questions/*",
+            "exclude": "SE1.0,*://*/questions/ask,*://*/questions*tab=*,*://chat.*.com/*"
         }, {
             "name": "unspoil",
             "desc": "Add a link to the bottom of a post to reveal all spoilers in a post",
-            "meta": "http://meta.stackexchange.com/questions/249808/add-a-way-to-reveal-all-spoiler-blocks-in-a-post-at-once",
-            "match": "*://*/questions*",
+            "meta": "https://meta.stackexchange.com/q/249808",
+            "match": "*://*/questions/*,*://*/questions*tab=*,*://chat.*.com/*",
```

Comment:
Do we want to match on `*://*/questions*tab=*` and `*://chat.*.com/*` for this feature, or should these be in the `exclude` property?